src/aliases/api/handlers/utils.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+)
+
+func jsonWrite(w http.ResponseWriter, data interface{}) error {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8;")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
+	return json.NewEncoder(w).Encode(data)
+}
+
+func WriteHTTPErrorResponse(rw http.ResponseWriter, err error) {
```

Comment:
We shouldn't duplicate this method (we already have the same in /stores/api/...`). Instead we should have all comoon http common stuff under `/src/infra/http` for reusability. `infra` is intended to encapsulate every implementation required to communicate (inbound and outbound) with external services and this includes http

Reply:
I understand that, but is it the point of this PR to move all this code into a common area and update code from stores to use the new common package?

Reply:
done already in another PR.